customer-plugin/src/main/java/org/entando/demo/customer/web/rest/AppUserResource.java

```diff
@@ -143,4 +155,44 @@ public AppUserResource() {
 
     }
 
-}
+    /**
+	 * {@code GET  /reset-password} : reset password for existing user.
+	 * 
+	 * @param email Id of existing user.
+	 * @return the {@link ForgotPasswordResponse} with status
+	 *         {@code 200 (if user exist and email sent)} and with
+	 *         ForgotPasswordResponse, or with status {@code 404 (Bad Request)} if
+	 *         the User does not exist.
+	 * @throws URISyntaxException if the Location URI syntax is incorrect.
+	 */
+	@CrossOrigin(origins = "*")
```

Comment:
This is an existing issue with this class since typically you do not want to open up services across domains. Generally this is managed in the SpringBoot application profile, not on a per method basis, but since this follows the pattern of the other method we'll leave it as-is.